RELEASE-NOTES.txt

```diff
@@ -3,6 +3,8 @@
  
 12.8
 -----
+* Setting featured image on a post doesn't require network connection
+Add percentage bar to following Stats cards - Posts and Pages, Authors, Tags and Categories
```

Comment:
Looks like there was a merge problem here.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10079/commits/709fd35f7401e55810e1e7c0ee944356f8c748f4